docs/tizen-studio/native-tools/ui-builder-n.md

```diff
@@ -4,9 +4,9 @@
 - Tizen Studio 1.0 and Higher
 
 
-The Tizen native UI Builder is a graphical user interface builder tool that simplifies the creation of Tizen native application UIs by allowing you to arrange UI components using drag-and-drop operations.
+The Tizen native UI Builder is a graphical user interface builder tool that simplifies the creation of Tizen native application UIs by allowing you to arrange UI components using drag-and-drop operations. You can also create [customized UI components](ui-builder-customcomponent.md) for application-specific purposes.
 
-The native UI Builder is not only a UI layout code generating tool. It also supports an easy-to-use programming model, allows you to [support multiple screens](multiple-screen-n.md) within a single application, and makes it possible to [manage various data sources through databinding](ui-builder-databind-n.md). If you have an existing EFL UI project, you can [convert it to a UI Builder project](ui-builder-conversion-n.md) to ease the task of expanding the application and optimizing it to new devices.
+The native UI Builder is not only a UI layout code generating tool. It also supports an easy-to-use programming model, allows you to [support multiple screens](multiple-screen_n.md) within a single application, and makes it possible to [manage various data sources through databinding](ui-builder-databind-n.md). If you have an existing EFL UI project, you can [convert it to a UI Builder project](ui-builder-conversion-n.md) to ease the task of expanding the application and optimizing it to new devices.
```

Comment:
Why did you change multiple-screen-n.md?

Reply:
It should be my mistake. I'll fix it.